### DISCUSSION TITLE
[Question] call_pyarmor && 8.2

### DISCUSSION THREAD

**Discussion by VeNoMouS** [dashingsoft/pyarmor#1260] [Q&A]
How do you use call_pyarmor with 8.2 as most of the new stuff isnt in there.. such as bcc etc..

**Comment by VeNoMouS** [comment]
How do you --bootstrap with `gen`? so the runtime isnt generated to another directory?

**Comment by VeNoMouS** [comment]
How do you enable `plugin`  with `8.20` for assert functions in code?

**Comment by jondy** [comment]
Please check 8.2 documentation.

**Comment by VeNoMouS** [comment]
> Please check 8.2 documentation.

i dont see any reference to `call_pyarmor` in the latest doco

**Comment by jondy** [comment]
https://pyarmor.readthedocs.io/en/stable/tutorial/installation.html#run-pyarmor-from-python-script

**Comment by VeNoMouS** [comment]
Cheers, i'll try have a play and see if i can replicate what i was doing in 7.x
